pkg/cache/cluster.go

```diff
@@ -167,6 +217,67 @@ type clusterCache struct {
 	eventHandlers               map[uint64]OnEventHandler
 }
 
+func (c *clusterCache) StoreApisMeta(key schema.GroupKind, value *apiMeta) {
+	c.apisMeta.Store(key, value)
+}
+
+func (c *clusterCache) LoadApisMeta(key schema.GroupKind) (*apiMeta, error) {
```

Comment:
Same as above - panicking should be ok if something unexpected is in this private field.